stellarphot/notebooks/photometry/06-transit-fit-template.ipynb

```diff
@@ -45,10 +45,17 @@
    "source": [
     "fo = FitsOpener(title=\"Select your photometry/flux file\", filter_pattern=[\"*.csv\", \"*.fits\", \"*.ecsv\"])\n",
     "fo2 = FitsOpener(title=\"Select your TESS info file\", filter_pattern=[\"*.pickle\"])\n",
+    "passband = ipw.Dropdown(description=\"Filter\", options=['gp', 'ip'], disabled=True)\n",
+    "\n",
+    "def update_filter_list(change):\n",
+    "    tab = Table.read(fo.path)['filter']\n",
+    "    passband.options = sorted(set(tab))\n",
+    "    passband.disabled = False\n",
+    "    passband.value = passband.options[0]\n",
```

Comment:
I assume this now adds the passbands in the Table automatically to the dropdown.